gulp/locales.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var gulp   = require('gulp');
+var rename = require('gulp-rename');
```

Comment:
Is rename accessible through this?
```
var $ = require('gulp-load-plugins')();

$.rename()
```
I can see in the mnoe-angular `gulp/inject.js` the script is accessing libs via `$`, but it has also defined the rename lib to a variable, so not sure whether this is done for a reason. If not, could be refactored there too.